src/modules/installers/react-web/actions.ts

```diff
@@ -181,25 +185,61 @@ export async function installSentry(): Promise<void> {
   await spinner.start();
 }
 
-export async function installComponent(component: i.Components): Promise<void> {
+export async function installComponent(component: string): Promise<void> {
   async function action() {
     const { projectName } = state.answers;
 
-    // Add dependencies without installing
-    const componentPath = `./prime-monorepo/components/web-components/${component}`;
-    if (existsSync(`${componentPath}/package.json`)) {
-      const { json: pkg } = await getPackageJson(
-        `${componentPath}/package.json`,
+    const installAndCopyComponent = async (
+      path: string,
+      destFolder: string,
+    ): Promise<string[] | null> => {
+      // If the component folder already exists (previously installed, skip)
+      if (existsSync(destFolder)) return null;
+
+      let peerDependencies = null;
+      // Read component package.json and add dependencies to project
+      if (existsSync(`${path}/package.json`)) {
+        const { json: pkg } = await getPackageJson(`${path}/package.json`);
+
+        // Save extra Label A components, so these can also be installed
+        const { labelaDependencies } = await addDependenciesFromPackage(pkg);
+        peerDependencies = labelaDependencies;
+      }
+
+      // Create component folder and copy /src folder from monorepo
+      await asyncExec(
+        `mkdir -p ${destFolder} && cp -r -n ${path}/src/. ${destFolder}`,
       );
 
-      await addDependenciesFromPackage(pkg);
-    }
+      // Rename component Storybook resolvers to valid project resolvers
+      await renameStorybookResolvers(destFolder);
```

Comment:
Betekent dit dat ook storybook files worden geinstalleerd?

Reply:
Nee, dit betekend het volgende:

Zoals je aan de linkerkant kun zien bij DatePicker.tsx heb ik de componenten nu opgedeeld in een root. Deze bevat `package.json`, `*.stories.tsx` en `code.ts` (met Storybook code examples). 

Alleen de `src` wordt gekopieerd naar je nieuwe project. Maar omdat met deze `src` folder de link naar de files niet meer klopte, heb ik het als volgt opgelost:

- Internal links `import { FormField } from '../FormField'` vervangen door een tsconfig resolver in het Storybook project `import { FormField } from '@labela/components/form/FormField/src';`
- Aan de rechterkant van de screenshot kun je ook zien dat in de `package.json` van de DatePicker de dependencies `"@labela/components/form/FormField": "1.0.0",` is. 
- `installAndCopyComponent` in `react-web/actions.ts > installComponent()` pakt deze component dependencies er zorgt er voor dat die ook gekopieerd en geïnstalleerd worden.
- `renameStorybookResolvers` zorgt er voor dat de Storybook TS resolvers worden vervangen door resolvers die je nieuwe project snapt `@labela/components/form/FormField/src` > `common/form/FormField`

Ben benieuwd hoe je hier over denkt :) 

<img width="2560" alt="Screenshot 2022-04-08 at 12 41 21" src="https://user-images.githubusercontent.com/4519142/162420136-60195eed-3495-4e6c-9618-af9fdae8a56a.png">

Reply:
Goeie! Zo laten dan